moped-editor/src/queries/project.js

```diff
@@ -783,7 +783,7 @@ export const PROJECT_UPDATE_LEAD = gql`
 `;
 
 export const PROJECT_UPDATE_PUBLIC_PROCESS = gql`
-  mutation ProjectUpdatePublicProcess($projectId: Int!, $fieldValueId: Int!) {
+  mutation ProjectUpdatePublicProcess($projectId: Int!, $fieldValueId: Int) {
```

Comment:
this is a bug Tilly found while testing her PR https://github.com/cityofaustin/atd-moped/pull/1308#issuecomment-2087798693